core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -62,10 +62,16 @@
  * <LI><b>readmodifywriteproportion</b>: what proportion of operations should be read a record,
  * modify it, write it back (default: 0)
  * <LI><b>requestdistribution</b>: what distribution should be used to select the records to operate
- * on - uniform, zipfian, hotspot, ordered or latest (default: uniform)
+ * on - uniform, zipfian, hotspot, ordered, exponential or latest (default: uniform)
  * <LI><b>maxscanlength</b>: for scans, what is the maximum number of records to scan (default: 1000)
  * <LI><b>scanlengthdistribution</b>: for scans, what distribution should be used to choose the
  * number of records to scan, for each scan, between 1 and maxscanlength (default: uniform)
+ * <LI><b>insertstart</b>: for parallel loads and runs, defines the starting record for this
+ * YCSB instance (default: 0)
+ * <LI><b>insertcount</b>: for parallel loads and runs, defines the ending record for this
```

Comment:
So, an instance will start at 'insertstart' record and go for 'insertcount' inserts? Is 'ending record' the right way to describe this? Its not exactly the ending record... but a count? This is a nit.